### PR TITLE
Expunge acwi

### PR DESCRIPTION
**Pull request by ayan-usgs**
Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [x] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

**Comment by jkreft-usgs** [comment]
lets hold of on merging this for the moment

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased](https://github.com/NWQMC/WQP_UI/compare/WQP_UI-5.20.0...master)
+## Changed
+- Purged logos and text pertaining to ACWI and/or NWQMC.
```
Comment (mbucknell) [inline CHANGELOG.md]: Causing Codacy issue, needs two more spaces.

Comment (ayan-usgs) [reply CHANGELOG.md]: Fixed.

**Comment by jkreft-usgs** [comment]
The reason to hold off is that it is not clear if we are supposed to drop the NWQMC logo quite yet

**Comment by ayan-usgs** [comment]
Overcome by events.

**Pull request closed without merging**